inspire_disambiguation/api.py

```diff
@@ -151,6 +176,15 @@ def cluster(ethnicity_model_path, distance_model_path, n_jobs, signature_block=N
     clusterer.fit(n_jobs=n_jobs)
     fit_time = datetime.now()
     for phonetic_block, cluster in clusterer.clusterer.clusterers_.items():
+        if test_signatures_uuids:
+            training_statistics = cluster.score(test_signatures_uuids, test_labels)
+            B3_f_score_all, wrongly_classified_pairs, B3_f_score_training, B3_f_score_validation = \
+                training_statistics[0][0], training_statistics[0][1],\
+                training_statistics[1], training_statistics[2]
```

Comment:
you should read about tuple unpacking:
```suggestion
            (B3_f_score_all, wrongly_classified_pairs), B3_f_score_training, B3_f_score_validation = training_statistics
```
but why this weird grouping?